target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -411,6 +411,7 @@ ath79_setup_interfaces()
 			"0@eth0" "2:lan:3" "3:lan:2" "4:lan:1" "5:lan:4"
 		;;
 	tplink,tl-wr842n-v2|\
+	tplink,tl-wr841hp-v2|\
 	tplink,tl-wr941hp-v1)
 		ucidef_set_interface_wan "eth1"
 		ucidef_add_switch "switch0" \
```

Comment:
are you sure the port order is correct ?

Reply:
Port order is confirmed. it has taken 2 days to figure out and i have found a exact match of ports with tl-wr842n-v2.